usage/rest-api/src/main/java/brooklyn/rest/api/ServerApi.java

```diff
@@ -87,6 +88,11 @@ public void shutdown(
             multiValueResponse = false)
     public String getStatus();
 
+    @GET
+    @Path("/getConfig/{configKey}")
```

Comment:
Would be better if this was `/config/{configKey}`, which would more closely match `EntityConfigApi` - that uses `/v1/applications/{application}/entities/{entity}/config/{config}` for retrieving a particular config value of a given entity.